### ISSUE TITLE
Drop-in 3.5.1 Crash - com.adyen.checkout.card.CardComponent.validateSecurityCode

### ISSUE THREAD

**Issue by tinsukE** [Adyen/adyen-android#152]
We are getting crash reports with the following stack trace:
```
Fatal Exception: java.lang.IndexOutOfBoundsException: Index: 0, Size: 1
       at java.util.ArrayList.get(ArrayList.java:437)
       at com.adyen.checkout.card.CardComponent.validateSecurityCode(CardComponent.java:270)
       at com.adyen.checkout.card.CardComponent.onInputDataChanged(CardComponent.java:131)
       at com.adyen.checkout.card.CardComponent.onInputDataChanged(CardComponent.java:41)
       at com.adyen.checkout.base.component.BasePaymentComponent.inputDataChanged(BasePaymentComponent.java:97)
       at com.adyen.checkout.card.CardView.notifyInputDataChanged(CardView.java:231)
       at com.adyen.checkout.card.CardView.access$200(CardView.java:45)
       at com.adyen.checkout.card.CardView$1.onTextChanged(CardView.java:94)
       at com.adyen.checkout.base.ui.view.AdyenTextInputEditText.afterTextChanged(AdyenTextInputEditText.java:56)
       at com.adyen.checkout.card.ui.CardNumberInput.afterTextChanged(CardNumberInput.java:61)
       at com.adyen.checkout.base.ui.view.AdyenTextInputEditText$1.afterTextChanged(AdyenTextInputEditText.java:82)
       at android.widget.TextView.sendAfterTextChanged(TextView.java:9802)
       at android.widget.TextView.setText(TextView.java:5771)
       at android.widget.TextView.autofill(TextView.java:10781)
       at android.view.autofill.AutofillManager.autofill(AutofillManager.java:1858)
       at android.view.autofill.AutofillManager.access$1700(AutofillManager.java:141)
       at android.view.autofill.AutofillManager$AutofillManagerClient.lambda$autofill$1(AutofillManager.java:2752)
       at android.view.autofill.-$$Lambda$AutofillManager$AutofillManagerClient$1jAzMluMSJksx55SMUQn4BKB2Ng.run(-.java:8)
       at android.os.Handler.handleCallback(Handler.java:873)
       at android.os.Handler.dispatchMessage(Handler.java:99)
       at android.os.Looper.loop(Looper.java:193)
       at android.app.ActivityThread.main(ActivityThread.java:6898)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:537)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
```
We don't have information about the payments themselves, but let me know if you need more information.

**Comment by caiofaustino** [comment]
Weird `Index: 0, Size: 1`, if size is 1 I'd expect the index 0 to be the one with the value.
I'll make a small change here that should take care of a possible asynchronous issue, but it's a bit of a shot in the dark. It will be in the next release but please let me know if you encounter this again.

**Comment by caiofaustino** [comment]
Fix released on `3.6.1`, please reopen the issue if you still have problems.
